native-vm-just-run-ninja/.gdb_history

```diff
@@ -0,0 +1,12 @@
+file ./out 
```

Comment:
Not sure all these files should be committed to git.

Reply:
It shouldn't. My mistake